### DISCUSSION TITLE
Github copilot student "your subscription has ended."

### DISCUSSION THREAD

**Discussion by Rufluenza** [community/community#33817] [Copilot Conversations]
I had signed up for the 60 day trial before being a student but that has ended.
the student version worked for a few days but now it doesn't.
I am still a student however my github copilot student has ended. I dont know what to do as i cannot sign up again. If anyone knows what to do please reply.

**Comment by iamcjwilliams** [comment]
@Rufluenza The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted. Also here is more info on [GitHub Global Campus ](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)
